src/js/helpers/web3.js

```diff
@@ -253,8 +330,25 @@ export function getAccountBalance() {
   })
 }
 
+export function getContractMethod({ contractObj, methodName, overloadKey }) {
+  return state.legacyWeb3
```

Comment:
It can take me a while to internalise nested ternary, I reckon it might be quicker for new eyes to understand what's happening if it was done with conditionals instead?